dashboard/static/js/analytics.js

```diff
@@ -1,3 +1,17 @@
+var ga_id = document.getElementById("push-dashboard").dataset.ga_id;
+var ga_filename = document.getElementById("push-dashboard").dataset.ga_filename;
```

Comment:
Yeah, looks like 97% of global browsers support it, and we know from MDN traffic that devs are a Chrome- and Firefox-heavy audience. So let's leave it for now and let users file an issue if it's hurting them.